docs/buildah-bud.md

```diff
@@ -48,6 +48,24 @@ resulting image's configuration.
 
 Images to utilise as potential cache sources. Buildah does not currently support caching so this is a NOOP.
 
+**--cap-add**=*CAP\_xxx*
+
+When executing RUN instructions, run the command specified in the instruction
+with the specified capability added to its capability set.
+Certain capabilities are granted by default; this option can be used to add
+more.
+
+**--cap-drop**=*CAP\_xxx*
+
+When executing RUN instructions, run the command specified in the instruction
+with the specified capability removed from its capability set.
+Certain capabilities are granted by default; this option can be used to remove
+them.
+
+If a capability is specified to both the **--cap-add** and **--cap-drop**
+options, it will be dropped, regardless of the order in which the options were
+given.
+
```

Comment:
Will need to add these to the podman build man page too, note to self.

Reply:
yes. I would love to see us add this list to inspect also, not in this PR though.